src/olympia/addons/tests/test_views.py

```diff
@@ -1996,6 +1996,14 @@ def test_set_slug_denied(self):
         )
         assert response.status_code == 200, response.content
 
+        # get the most recent ADDON_SLUG_CHANGED log entry
+        log_entry = ActivityLog.objects.filter(
+            action=amo.LOG.ADDON_SLUG_CHANGED.id
+        ).latest('pk')
+        assert log_entry.user == self.user
+        assert log_entry.arguments == [self.addon, 'denied-slug', 'denied-slug~']
+        assert 'slug from denied-slug to denied-slug~' in str(log_entry)
```

Comment:
Let's move that to a completely separate test - we don't want to lose coverage of this if we ever refactor away the denied slug logic.